### PR TITLE
docs: futher polishing

### PR DESCRIPTION
**Pull request by mihai-sysbio**
### Main improvements in this PR:

This PR is used to highlight a few places where the tutorials can be further improved/simplified. Please feel free to push commits to this branch.

**I hereby confirm that I have:**

- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

**Comment by mihai-sysbio** [comment]
It takes less than 10 min to run the light tutorial now, great job 🌟 (I know it's using a lot of pre-computed values in the TSV files, but we do provide ways to reproduce them directly in the tutorial)

**Comment by mihai-sysbio** [comment]
The order of installing GECKO and checking the RAVEN installation is different in the 2 tutorials. Any insight into which is preferred?

**Comment by mihai-sysbio** [comment]
In STEP 22 of the full tutorial
> sol = solveLP(ecModel)

prints a growth rate of 0.0887, which very close to 0.0889 but still different. Any reactions?

**Comment by mihai-sysbio** [comment]
I've ran both tutorials several times in less than 1 hour in total, very convenient 🤩

**Comment by edkerk** [comment]
> The order of installing GECKO and checking the RAVEN installation is different in the 2 tutorials. Any insight into which is preferred?

Best to have RAVEN first, as GECKO will do a check that RAVEN can be found, I'll swap it.

> In STEP 22 of the full tutorial
> 
> > sol = solveLP(ecModel)
> 
> prints a growth rate of 0.0887, which very close to 0.0889 but still different. Any reactions?

I'll slightly rephrase it to avoid mentioning precise numbers.

**Review by edkerk — APPROVED** [review]
